crates/gen-rust/src/lib.rs

```diff
@@ -284,6 +285,9 @@ pub trait RustGenerator {
             TypeDefKind::Union(_) => {
                 panic!("unsupported anonymous type reference: union")
             }
+            TypeDefKind::Stream(_) => {
+                panic!("unsupported anonymous type reference: stream")
```

Comment:
I think that this will need to be supported so I think this will need to be `todo!`